### PR TITLE
added solution to lc:1251

### PR DESCRIPTION
**Pull request by Dhoni77**
✅ Updated readme
✅ Added solution file

### CODE REVIEW

File: solution/1200-1299/1251.Average Selling Price/README.md [hunk 1]
```diff
@@ -83,7 +83,14 @@ Result table:
 ### **SQL**
 
 ```sql
-
+SELECT p.product_id,
+       Round(( Sum(u.units * p.price) + 0.0 ) / ( Sum(units) + 0.0 ), 2)
+       avg_price
```
Comment (yanglbme) [inline solution/1200-1299/1251.Average Selling Price/README.md:88]:
```suggestion
       average_price
```

Comment (yanglbme) [reply solution/1200-1299/1251.Average Selling Price/README.md:88]: Write an SQL query to find the average selling price for each product. `average_price` should be rounded to 2 decimal places.

**Review by yanglbme — CHANGES_REQUESTED** [review]

File: solution/1200-1299/1251.Average Selling Price/README.md (hunk 1 above)
Comment (Dhoni77) [reply solution/1200-1299/1251.Average Selling Price/README.md]: Done ✅

**Review by yanglbme — APPROVED** [review]

**Comment by Dhoni77** [comment]
Thanks @yanglbme 👍

**Pull request merged**